packages/listview/src/components/AuditLogs/filterDropdown/filters.tsx

```diff
@@ -32,20 +33,29 @@ const LabelFilter = ({
     index: number;
     label: string;
     filteredLabels: string[];
-    onChange: (oldName: string, newName?: string, value?: string) => void;
+    onChange: OnChangeFunction;
 }) => {
     const { labels } = useContext(context);
     const inputRef = useRef<HTMLInputElement>();
 
     const handleNameChange = ({ name }: PicklistValue) =>
-        onChange(nameInProps, name as string, valueInProps);
+        onChange({
+            oldName: nameInProps,
+            newName: name as string,
+            value: valueInProps,
+        });
 
     const handleValueChange = (event: React.ChangeEvent<HTMLInputElement>) => {
         const newValue = event.target.value;
-        onChange(nameInProps, nameInProps, newValue);
+        onChange({
+            oldName: nameInProps,
+            newName: nameInProps,
+            value: newValue,
+        });
```

Comment:
@LeandroTorresSicilia this is intentional. The onChange function updates the input value when the `oldName` and `newName` are equal.